src/sentry/api/endpoints/release_threshold_details.py

```diff
@@ -10,8 +10,7 @@
 from sentry.api.bases.project import ProjectEndpoint
 from sentry.api.exceptions import ResourceDoesNotExist
 from sentry.api.serializers import serialize
-from sentry.models.project import Project
-from sentry.models.release_threshold.releasethreshold import ReleaseThreshold
```

Comment:
where do we prefer importing models like this? i see both styles sometimes

Reply:
Specific imports are fine, however we're already exporting everything from the models/__init__.py so imo it's cleaner to simply have the top level path for imports.

Reply:
I think we're trying to get away from this like @JoshFerge pointed out

Reply:
Ah - sounds like we're actually pushing for the explicit imports so reverting and changing to that